### PR TITLE
explanation for debug mode

### PR DESCRIPTION
**Pull request by franzholz**
.user.ini:
TYPO3_CONTEXT = Development

### CODE REVIEW

File: Documentation/In-depth/TheInstallTool/Index.rst [hunk 1]
```diff
@@ -99,8 +99,22 @@ administrators. As administrators still have elevated permissions you should
 still be careful which users can be trusted with these rights.
 
 At **Configuration Presets** TYPO3 offers presets for common settings groups.
-It allows for easier configuration of settings for debugging, image handling,
+It allows for easier configuration of settings for caching, debugging, image handling,
```
Comment (DavidBruchmann) [inline Documentation/In-depth/TheInstallTool/Index.rst:102]: It allows easier configuration ...

File: Documentation/In-depth/TheInstallTool/Index.rst [hunk 2]
```diff
@@ -99,8 +99,22 @@ administrators. As administrators still have elevated permissions you should
...
 mail configuration and password hashing.
+The debug settings gives you 3 choices:
+
+#. Live
+   Turn off debug output and set logging to warnings and errors only.
+#. Debug
+   Enable debug output and set logging to info level.
+#. Custom configuration
+   Custom configuration mixture if no other preset fits.
```
Comment (DavidBruchmann) [inline Documentation/In-depth/TheInstallTool/Index.rst:111]: Custom configuration is a mixture of settings related to debugging.

File: Documentation/In-depth/TheInstallTool/Index.rst (hunk 1 above)
Comment (mbrodala) [reply Documentation/In-depth/TheInstallTool/Index.rst:102]: Technically _allow for_ is correct here: https://www.merriam-webster.com/dictionary/allow%20for

File: Documentation/In-depth/TheInstallTool/Index.rst [hunk 3]
```diff
@@ -99,8 +99,22 @@ administrators. As administrators still have elevated permissions you should
...
+   
+The Debug option will activate a debug output with a multiline backtrace in the frontend, in the case of errors. 
+However it only works if the environment variable *TYPO3_CONTEXT* has been set to `Development`.
```
Comment (DavidBruchmann) [inline Documentation/In-depth/TheInstallTool/Index.rst:114]: Debug output is only displayed if the environment variable *TYPO3_CONTEXT* has been set to `Development`.

File: Documentation/In-depth/TheInstallTool/Index.rst [hunk 4]
```diff
@@ -175,7 +191,6 @@ This is the recommended way.
 
 **Test Mail Setup** allows you to test the mail setup of TYPO3 by sending a
 test mail.
-
```
Comment (mbrodala) [inline Documentation/In-depth/TheInstallTool/Index.rst:178]: This line should be kept.

File: Documentation/In-depth/TheInstallTool/Index.rst [hunk 5]
```diff
@@ -99,8 +99,24 @@ administrators. As administrators still have elevated permissions you should
...
+#. Live
+   Turn off debug output and set logging to warnings and errors only.
+#. Debug
+   Enable debug output and set logging to info level.
+#. Custom configuration
+   Custom configuration is a mixture of settings how to debug and report errors.
+   
+The Debug option will activate a debug output with a multiline backtrace in the frontend, 
+in case of errors. Debug output is only displayed if the environment variable 
+*TYPO3_CONTEXT* has been set to `Development`.
```
Comment (mbrodala) [inline Documentation/In-depth/TheInstallTool/Index.rst:115]:
```suggestion
`TYPO3_CONTEXT` has been set to `Development`. See :ref:`t3coreapi:application-context` for details.
```

**Review by mbrodala — CHANGES_REQUESTED** [review]

File: Documentation/In-depth/TheInstallTool/Index.rst (hunk 4 above)
Comment (franzholz) [reply Documentation/In-depth/TheInstallTool/Index.rst:178]: This line has not been rmoved
It is stil there:

> It offers you the possibility to create a file :file:`.htaccess` in the TYPO3 root directory.
> This is the recommended way.

Comment (mbrodala) [reply Documentation/In-depth/TheInstallTool/Index.rst:178]: Please have a closer look at the line I commented on. You seem to have dropped an empty line which should be reverted.

File: Documentation/In-depth/TheInstallTool/Index.rst [hunk 6]
```diff
@@ -192,4 +207,3 @@ and displays hints if something goes wrong.
    TYPO3 might have problems executing it. If the command does not work, you
    might need to adjust the settings (via "Settings" > "Configuration Presets"
    > "Image Handling").
-
```
Comment (mbrodala) [inline Documentation/In-depth/TheInstallTool/Index.rst:195]: Please keep the empty line at EOF

File: Documentation/In-depth/TheInstallTool/Index.rst (hunk 4 above)
Comment (franzholz) [reply Documentation/In-depth/TheInstallTool/Index.rst]: I did not notice that.

File: Documentation/In-depth/TheInstallTool/Index.rst (hunk 6 above)
Comment (linawolf) [reply Documentation/In-depth/TheInstallTool/Index.rst:195]: @mbrodala if you hit the edit mode there is an empty line at the end of the file. Somehow this sometimes appears in PRs, but the empty line at the end is automatically added so it didn't disapear.

**Review by franzholz — COMMENTED** [review]
ok

**Pull request merged**